### ISSUE TITLE
News Feed

### ISSUE THREAD

**Issue by jaseiler** [researchcompendia/researchcompendia#87]
A news feed with small (1-3 lines) items for updates on the site's progress and features and also news about the team, outreach, collaborations. Perhaps with links out to blog posts for some items. 

A simple quick (temporary) solution would be to use the twitter feed tools to just feed in the ResearchCompendia tweets. That would be super easy. (see: http://jennseiler.com/#forum)

If items are small enough perhaps it could be a sidebar on the home page. Or it could be below, and a calendar could be a sidebar (if we ever get enough calendar items).

It might be nice to have a separate feed for newest compendia, but I think just being able to browse by creation date on the browse page should suffice for that.

**Comment by codersquid** [comment]
It'll be quick to add a twitter timeline so I'll do that today and have it up on preprod.

**Comment by codersquid** [comment]
It's up on http://labs.researchcompendia.org/ now. The researchcompend timeline doesn't have any tweets and the page layout looks horrible with no tweets. I made a list for now with four members.

https://twitter.com/codersquid/researchcompendia/members

**Comment by jaseiler** [comment]
I like it, but I want to be sure the about box fits vertically in most browser windows.  Could we lessen that massive top margin above the words "Research Compendia" (and the bottom margin to match) and maybe make the text a bit smaller in that box?

**Comment by jaseiler** [comment]
PS: could you make me an admin on the preprod site?

**Comment by codersquid** [comment]
Gave you admin, sorry for missing that!

I agree I don't like how much space the top and bottom margin takes up. I'll adjust the style.

**Comment by codersquid** [comment]
I made some changes and pushed them to labs.researchcompendia.org -- I reduced the brand size from 18px to 14px, added a dropdown nav item with menu items from the bottom navbar, and I unfixed the bottom navbar.

Does that provide more space in a reasonably pleasant way.

**Comment by jaseiler** [comment]
Please leave the bottom bar fixed.  I'd really rather just make the margins smaller in the about box.  But worst case, I'd rather have the box go over than lose the permanent visibility of the bottom bar .

**Comment by codersquid** [comment]
Is the about box what I've been thinking of as the jumbotron? I realize
I've been misunderstanding your review comment now.

On Tue, Dec 10, 2013 at 7:35 PM, Jennifer Seiler
notifications@github.comwrote:

> Please leave the bottom bar fixed. I'd really rather just make the margins
> smaller in the about box. But worst case, I'd rather have the box go over
> than lose the permanent visibility of the bottom bar .
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/researchcompendia/tyler/issues/87#issuecomment-30286565
> .

## 

sheila@codersquid.com

**Comment by jaseiler** [comment]
Yes, the jumbotron, the top and bottom padding on the text is too big inside the jumbotron.  The bottom bar needs to be fixed, even the analytics say that's how everyone is navigating around.

**Comment by codersquid** [comment]
Our analytics will show that due to a chicken/egg problem. We could try some A/B testing at some point to test different placement scenarios.

Try again http://labs.researchcompendia.org/ I've add .rc-jumbotron to our project.css. It shrinks padding and font size.

**Comment by jaseiler** [comment]
What happened to the news feed??

**Comment by codersquid** [comment]
Other issues got deployed to prepod over it so that they could be tested.
It's not accepted yet, so it's not merged yet.

**Comment by codersquid** [comment]
I'm breaking this out in to two different issues -- this one for the news feed, another for the style changes.
